Include/internal/pycore_cell.h

```diff
@@ -42,6 +43,31 @@ PyCell_GetRef(PyCellObject *cell)
     return res;
 }
 
+static inline void
+_PyCell_GetStackRef(PyCellObject *cell, _PyStackRef *value_addr)
```

Comment:
```suggestion
_PyStackRef _PyCell_GetStackRef(PyCellObject *cell)
```
I think that this can return the new `_PyStackRef` directly instead of via pointer, since there can be no garbage collection in this function.